<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputSize</class>
 <widget class="QMainWindow" name="InputSize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>185</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>365</width>
    <height>185</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>365</width>
    <height>185</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Input size - Maze Solver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>142</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vBoxLayout_main">
     <item>
      <widget class="QLabel" name="label_title">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>17</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Input maze size : </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hBoxLayout_height">
       <item>
        <widget class="QLabel" name="label_height">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Height : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_height">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hBoxLayout_width">
       <item>
        <widget class="QLabel" name="label_width">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Width : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_width">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar_message"/>
 </widget>
 <resources/>
 <connections/>
</ui>
